frontend/ui/data/onetime_migration.lua

```diff
@@ -888,6 +888,28 @@ if last_migration_date < 20250601 then
     SettingsMigration:migrateSettings(G_reader_settings)
 end
 
+-- 20250629, Fix trailing slashes in wallabag directories #13987
+-- https://github.com/koreader/koreader/issues/13987
+if last_migration_date < 20250629 then
+    logger.info("Performing one-time migration for 20250629")
+
+    local wb_lua = DataStorage:getSettingsDir() .. "/wallabag.lua"
+    if lfs.attributes(wb_lua, "mode") == "file" then
+        local wb_settings = LuaSettings:open(wb_lua)
+        wb_settings:readSetting("wallabag")
```

Comment:
If you do
```Lua
local settings = wb_settings:readSetting("wallabag")
```
you can use shorter `settings.directory` and avoid `saveSettings` later.